main.go

```diff
@@ -213,7 +191,6 @@ func parseCoverage(coverage io.Reader) (map[string][]*block, error) {
 				log.Printf("warn: %v", err)
 				continue
 			}
-			f = getCoverallsSourceFileName(f)
```

Comment:
pass this function as a parameter. Default is to pass `getCoverallsSourceFileName`, if `-use-absolute-source-path` is set, just pass identify, `func identity(f) string {return f;}`